sender.go

```diff
@@ -151,10 +158,15 @@ func (s *Sender) send(ctx context.Context, msg *Message) (chan encoding.Delivery
 
 		select {
```

Comment:
I'm not sure if this is a deadly situation - if the user fails to transfer a frame they can always call and transfer it again, correct? If it's inconsistent at that point we'll get an errror from the broker but I don't think we need to preemptively kill it here.

Reply:
I have reverted this for now.  However, in a multi-part transfer, if we fail to send the Nth frame in, how do we handle it (I added a comment about this)?